firmwares/ultraloq/U-BOLT-PRO-ZWAVE.json

```diff
@@ -0,0 +1,25 @@
+{
+	"devices": [
+		{
+			"brand": "Ultraloq",
+			"model": "U-BOLT-PRO-ZWAVE",
+			"manufacturerId": "0x0452",
+			"productType": "0x0004",
+			"productId": "0x0001"
+		}
+	],
+	"upgrades": [
+		{
+			"version": "1.5",
+			"channel": "beta",
+			"changelog": "Fixed door state issue",
+			"files": [
+				{
+					"target": 0,
+					"url": "https://file.u-tec.com/homekit/U-BOLT-PRO-ZWAVE_V1.5.gbl",
+					"integrity": "sha256:51161b4e2dd2607ab246782fd267022c30811fb6a9f233ff451e006d9a07179a"
```

Comment:
This seems to be wrong. Both these commands:
```
npx @zwave-js/firmware-integrity@latest https://file.u-tec.com/homekit/U-BOLT-PRO-ZWAVE_V1.5.gbl
```
and
```
curl https://file.u-tec.com/homekit/U-BOLT-PRO-ZWAVE_V1.5.gbl | sha256sum
```

give me a different integrity hash
```
sha256:fc0b831cf69dfbf4af7b6c95ee92b349899390b36daedb3413bfa17117eb4bab
```

I'll fix it, assuming this was a mistake.